src/app/providers/sessionRefreshProvider.tsx

```diff
@@ -19,5 +31,9 @@ export const SessionRefreshProvider = ({ children }: Props) => {
     }
   }, [session]);
 
-  return <>{children}</>;
+  return (
+    <>
+      <SplitsProvider config={splitsConfig}>{children}</SplitsProvider>
+    </>
+  );
```

Comment:
This PR is missing the whole setup of checking the wallet client load statuses;

Don't forget to add the 
`<>{children}</>`
on the scenario that it has not been loaded yet 👍